src/clientfactory.ts

```diff
@@ -113,7 +116,7 @@ export class DiscordClientFactory {
         client.on("warn", (msg) => { jsLog.warn(msg); });
 
         try {
-            await client.login(token, false);
+            await client.login(token);
```

Comment:
I *believe* discord.js doesn't support logging in as a user account because it's against Discord's TOS, so I'm not so sure that part of the code works as intended, whatever the intentions are in the first place